clients/src/main/java/org/apache/kafka/common/security/ssl/SslFactory.java

```diff
@@ -64,8 +76,13 @@ public SslFactory(Mode mode) {
     }
 
     public SslFactory(Mode mode, String clientAuthConfigOverride) {
+        this(mode, clientAuthConfigOverride, false);
+    }
+
+    public SslFactory(Mode mode, String clientAuthConfigOverride, boolean clientAndServer) {
```

Comment:
I meant that the configs can be used for both client-side and server-side, but agree that the name is confusing. Renamed to `keystoreVerifiableUsingTruststore` since that is what we are actually verifying.